Cargo.toml

```diff
@@ -25,6 +25,7 @@ generic-array = "0.14.5"
 hkdf = "0.12.3"
 hyp_alloc = { path = "./hyp-alloc" }
 memoffset = { version = ">=0.6.5", features = ["unstable_const"] }
+mtt_management = { path = "./mtt-management" }
```

Comment:
It will be consumed as part of the page conversion flows, so modelling it after riscv_pages etc. seems to make sense. That said, I don't really feel strongly one way or the other, so am open to suggestions (same holds for the nomenclature as well).

Reply:
That makes sense. No strong opinion either, just curious of what this will be used for. Thanks!